fmriprep/workflows/anatomical.py

```diff
@@ -3,9 +3,25 @@
 # emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
 # vi: set ft=python sts=4 ts=4 sw=4 et:
 """
-Anatomical Reference -processing workflows.
+Anatomical reference preprocessing workflows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Originally coded by Craig Moodie. Refactored by the CRN Developers.
+.. autofunction:: init_anat_preproc_wf
+.. autofunction:: init_skullstrip_ants_wf
+
+Surface preprocessing
++++++++++++++++++++++
+
+``fmriprep`` uses FreeSurfer_ to reconstruct surfaces from T1w/T2w
+structural images.
+
+.. autofunction:: init_surface_recon_wf
+.. autofunction:: init_autorecon_resume_wf
+.. autofunction:: init_gifti_surface_wf
+
+  .. note ::
```

Comment:
This note appears just below the gifti_surface workflow and that's misleading. Please move it to `init_anat_preproc_wf` (or maybe we should just write some acknowledgment in the About FMRIPREP and remove these notes from here and there).